src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/Helpers.cs

```diff
@@ -465,5 +465,20 @@ internal static void ThrowIfPasswordContainsNullCharacter(string? password)
 
             return array;
         }
+
+        internal static bool IsSlhDsaOid(string? oid) =>
+            oid is
+                Oids.SlhDsaSha2_128s or
+                Oids.SlhDsaShake128s or
+                Oids.SlhDsaSha2_128f or
+                Oids.SlhDsaShake128f or
+                Oids.SlhDsaSha2_192s or
+                Oids.SlhDsaShake192s or
+                Oids.SlhDsaSha2_192f or
+                Oids.SlhDsaShake192f or
+                Oids.SlhDsaSha2_256s or
+                Oids.SlhDsaShake256s or
+                Oids.SlhDsaSha2_256f or
+                Oids.SlhDsaShake256f;
```

Comment:
Until this is different from `SlhDsaAlgorithm.GetAlgorithmForOid(oid) is not null` it should be written in terms of that (so we don't have two OID lists when we don't need to).

Having the helper for just the booleanness is fine, "someFunc() is not null" checks do sometimes look weird.  But I'd implement this in terms of that.

Reply:
There's going to end up being duplication since SignedCms in S.S.C.Pkcs is also going to need it. Having `GetAlgorithmForOid` colocated with `SlhDsaAlgorithm` is nice but if we want to avoid the duplication we can move it out into this helper.